examples/pruning/keras_integration.py

```diff
@@ -0,0 +1,94 @@
+"""
+Optuna example that demonstrates a pruner for Keras.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+Keras and MNIST, where the architecture of the neural network is optimized. Throughout the
+training of neural networks, a pruner observes intermediate results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python keras_integration.py
+
+"""
+
+import optuna
+from optuna.integration import KerasPruningCallback
+
+import keras
+from keras.datasets import mnist
+from keras.layers import Dense
+from keras.models import Sequential
+from keras import optimizers
+from keras import regularizers
+
+
+BATCHSIZE = 128
+CLASSES = 10
+EPOCHS = 20
+
+
+def model_fn(trial):
+    # Model
+    n_layers = trial.suggest_int('n_layers', 1, 3)
+    weight_decay = trial.suggest_uniform('weight_decay', 1e-10, 1e-3)
+    model = Sequential()
+    for i in range(n_layers):
+        num_hidden = int(trial.suggest_loguniform('n_units_l{}'.format(i), 4, 128))
+        model.add(Dense(num_hidden,
+                        activation='relu',
+                        kernel_regularizer=regularizers.l2(weight_decay)))
+    model.add(Dense(CLASSES,
+                    activation='softmax',
+                    kernel_regularizer=regularizers.l2(weight_decay)))
+
+    # Optimizer
+    lr = trial.suggest_uniform('lr', 1e-5, 1e-1)
+    optimizer = trial.suggest_categorical('optimizer', ['RMSprop', 'Adam', 'SGD'])
+    model.compile(loss='categorical_crossentropy',
+                  optimizer=getattr(optimizers, optimizer)(lr=lr),
```

Comment:
I think some novice users of Optuna are not familiar with this notation, and they may have difficulty in reading this example. Could you fix the optimizer like [chainer example](https://github.com/pfnet/optuna/blob/master/examples/pruning/chainer_integration.py#L52) to simplify the example?
I think `RMSprop` is a first choice because [the official example](https://github.com/keras-team/keras/blob/master/examples/mnist_mlp.py#L46) in keras-team repository uses it.

Reply:
Alright.